app/src/main/java/com/locoquest/app/dao/IBenchmarkDAO.kt

```diff
@@ -12,6 +12,10 @@ import retrofit2.http.Query
 interface IBenchmarkDAO {
     @GET("pid")
     fun getBenchmarkByPid(@Query("pid") pid: String): Call<Benchmark>
+
+    @GET("/")
+    fun getAllBenchmarks(): Call<ArrayList<Benchmark>>
```

Comment:
This appears to be an incomplete feature, as it does not have an endpoint.

Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.

Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.